README.md

```diff
@@ -25,16 +19,15 @@ See [TESTPLAN.md](./docs/TESTPLAN.md) for more details on how to get the add-on
 ## Data Collected / Telemetry Pings
 
 Measure:
-
-* Button (BrowserAction) usage.
+- Button (BrowserAction) usage.
 
 See [TELEMETRY.md](./docs/TELEMETRY.md) for more details on what pings are sent by this add-on.
 
 ## Analyzing data
 
 Telemetry pings are loaded into S3 and re:dash. Sample query:
 
-* [All pings](https://sql.telemetry.mozilla.org/queries/{#your-id}/source#table)
+ * [All pings](https://sql.telemetry.mozilla.org/queries/{#your-id}/source#table)
```

Comment:
Will the `{#your-id}` looking placeholder/variable cause a 404 here?

Reply:
Yes. It is meant to be replaced in the template by the study developer.